src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/ListSavedRecordingsCommandTest.java

```diff
@@ -137,8 +137,46 @@ void shouldExecuteAndPrintSavedRecordings() throws Exception {
 
         InOrder inOrder = inOrder(cw);
         inOrder.verify(cw).println("Saved recordings:");
-        inOrder.verify(cw).println("\tfoo");
-        inOrder.verify(cw).println("\tbar");
+        inOrder.verify(cw).println(Mockito.contains("getName\t\tfoo"));
+        inOrder.verify(cw).println(Mockito.contains("getName\t\tbar"));
+    }
+
+    @Test
+    void shouldPrintURLs() throws Exception {
+        when(fs.listDirectoryChildren(recordingsPath)).thenReturn(Arrays.asList("foo", "bar"));
+        when(exporter.getArchivedDownloadURL(Mockito.anyString())).thenAnswer(
+            new Answer<String>() {
+                @Override
+                public String answer(InvocationOnMock invocation) throws Throwable {
+                    return String.format(
+                            "http://example.com:1234/api/v1/recordings/%s",
+                            invocation.getArguments()[0]);
+                }
+            });
+        when (exporter.getArchivedReportURL(Mockito.anyString())).thenAnswer(
```

Comment:
There's an extra whitespace after the `when` on this line. Have you done a `mvn spotless:apply` to auto-format the code?

Reply:
Fixed!